packages/solid/src/render/flow.ts

```diff
@@ -258,3 +259,23 @@ export function ErrorBoundary(props: {
     "_SOLID_DEV_" ? { name: "value" } : undefined
   ) as Accessor<JSX.Element>;
 }
+
+export function ClientOnly(props: {
+  children?: JSX.Element,
```

Comment:
Is there a use-case where you'd want to use a self-closing `<ClientOnly />`?

Reply:
Maybe 🤷 🤣

Reply:
If there is a play, there is a player 😆